docs/pages/kubernetes-access/manage-access/rbac.mdx

```diff
@@ -0,0 +1,427 @@
+and one or more Kubernetes clusters. 
+
+When a user authenticates to Teleport, they receive a kubeconfig that lets them
+send requests to to their authorized Kubernetes clusters via the Teleport
+Kubernetes Service. The Kubernetes Service can then inspect, modify, or disallow
+these requests depending on the privileges you have assigned to the Teleport
+user via their roles.
+
+In this guide, we will use a local Kubernetes cluster to show you how to
+configure Teleport's role-based access control (RBAC) system to manage access to
+Kubernetes clusers, groups, users, and resources.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+(!docs/pages/includes/tctl.mdx!)
+
+To run the local demo environment, ensure that you have the following tools
+installed on your workstation:
+
+| Tool                                                                | Purpose                          | Installation link                                             |
+|---------------------------------------------------------------------|----------------------------------|---------------------------------------------------------------|
+| minikube                                                            | Local Kubernetes deployment tool | [Install minikube](https://minikube.sigs.k8s.io/docs/start/)  |
+| Helm                                                                | Kubernetes package manager       | [Install Helm](https://helm.sh/docs/intro/install/)           |
+| kubectl                                                             | Kubernetes admin CLI             | [Install kubectl](https://kubernetes.io/docs/tasks/tools/)    |
+| Docker | Required minikube driver         | [Get Started With Docker](https://www.docker.com/get-started) |
+
+## Step 1/3. Prepare Kubernetes resources
+
+### Start minikube
+
+Start minikube with the Docker driver:
+
+```code
+$ minikube start --driver=docker
+```
+
+This command should start a local Kubernetes cluster and set your context to
+`minikube`. To verify this, run the following command:
+
+```code
+$ kubectl config current-context
+minikube
+```
+
+### Deploy demo pods
+
+On your workstation, create a manifest file called `pods.yaml` with the
+following content:
+
+```yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: development
+  labels:
+    name: development
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: production
+  labels:
+    name: production
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: webapp
+  namespace: development
+spec:
+  containers:
+  - name: nginx
+    image: nginx:1.23
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: webapp
+  namespace: production
+spec:
+  containers:
+  - name: nginx
+    image: nginx:1.23
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: loadbalancer
+  namespace: development
+spec:
+  containers:
+  - name: nginx
+    image: nginx:1.23
+---
+apiVersion: v1
```

Comment:
In this example one can be a deployment so you can show the full power of Pod RBAC using wildcards since Kubernetes does not allow that in RBAC